src/emailLogin/runMigration.js

```diff
@@ -45,6 +45,10 @@ const getSiteAndContributors = async (site, dbClient) => {
     const siteMemberValues = [];
     userData.forEach((user) => {
       const userId = user.id;
+      if (!user.email) {
+        // User not registered to an email
+        return;
+      }
       const userType = whitelistedDomains.filter((domain) => user.email.endsWith(domain)).length > 0 ? 'ADMIN' : 'CONTRIBUTOR';
```

Comment:
Hey this should be `user.email.endsWith(domain))` rather than `user.email.endsWith(domain)).length > 0` right?

Reply:
It's actually `whitelistedDomains.filter((domain) => user.email.endsWith(domain))` and then checking that at least one of the whitelist domains match (`.length > 0`)

Reply:
arh HAHA ok makes sense